git-archive-all

```diff
@@ -13,6 +38,8 @@ import sys
 import tarfile
 from zipfile import ZipFile, ZipInfo, ZIP_DEFLATED
 
+__version__ = "1.12"
+
```

Comment:
imports needs to be at the beginning of the file ( before module globals and constants ) https://www.python.org/dev/peps/pep-0008/#imports